app/database/models/member.py

```diff
@@ -2,7 +2,14 @@
 
 
 class Member(Document):
-    gid   = LongField(required=True)
-    uid   = LongField(required=True, primary_key=True, unique_with=["gid"])
-    xp    = LongField(required=True)
-    level = LongField(required=True)
+    gid           = LongField(null=False, required=True)
+    uid           = LongField(null=False, required=True, primary_key=True, unique_with=["gid"])
+    xp            = LongField(default=0)
+    level         = LongField(default=0)
+    display_name  = StringField()
+    name          = StringField()
+    discriminator = LongField()
+    joined_at     = DateTimeField()
+    lab_member_number = LongField()
```

Comment:
`lab_member_number` should probably be a `IntField` instead of `LongField`. I'm sure ~2-4 Million entries is enough (I'd propose `ShortField` but it doesn't look like it's a default type).

In the same vein (this isn't a you thing), but we can probably do the same for the `xp` and `level`. Since `xp` is the current xp between levels and not the total raw xp, we should be fine for a while (level 100 only requires 10 000 xp to rank up), and I doubt someone will hit level 2-4 million.

Reply:
Changes for `lab_member_number`. Do you want me to change for `xp` and `level` too?

Reply:
If you haven't pushed it yet, that would be nice, otherwise it can be changed in a later commit.

Reply:
Consider it done 🥼